backend/inspirehep/submissions/views.py

```diff
@@ -191,7 +200,34 @@ def load_data_from_request(self):
     def start_workflow_for_submission(self, record):
         record["acquisition_source"] = self.get_acquisition_source()
         payload = {"data": record}
-        return self.send_post_request_to_inspire_next("/workflows/authors", payload)
+
+        if current_app.config.get("FEATURE_FLAG_ENABLE_SEND_TO_BACKOFFICE"):
+            try:
+                payload_backoffice = {
+                    "data": record,
+                    "workflow_type": "AUTHOR_CREATE",
+                    "status": "running",
+                    "core": False,
+                    "is_update": True,
+                }
+                self.send_post_request_to_inspire_next(
+                    current_app.config["INSPIRE_BACKOFFICE_URL"],
+                    "/api/workflows/",
+                    payload_backoffice,
+                    current_app.config["BACKOFFICE_BEARER_TOKEN"],
+                    bearer_keyword="Token",
+                )
+            except WorkflowStartError:
+                print(
+                    f"Error creating record in {current_app.config['INSPIRE_BACKOFFICE_URL']} ",
+                    str(payload_backoffice),
+                )
```

Comment:
we don't need to catch this error. In general we don't use print, instead we log the errors because it will send them to monitoring OpenSearch